doc/source/user_guide/ug_case_settings.rst

```diff
@@ -49,14 +54,19 @@ to support the CICE model.
    "ICE_BASECOM", " ", "baseline directory name for regression comparison, associated with cice.setup -bcmp ", "set by cice.setup"
    "ICE_BFBCOMP", " ", "location of case for comparison, associated with cice.setup -td", "set by cice.setup"
    "ICE_SPVAL", " ", "special value for cice.settings strings", "set by cice.setup"
-   "ICE_RUNLENGTH", " ", "batch run length default", "set by cice.setup"
+   "ICE_RUNLENGTH", "integer", "batch run length default", "set by cice.setup"
+   " ", "0", "15 minutes (default)", " "
+   " ", "1", "59 minutes", " "
+   " ", "2", "2 hours", " "
+   " ", "other N$<$8", "N hours", " "
```

Comment:
Should be:
   " ", "other :math:`2 < N < 8`", "N hours", " "

Reply:
The block of math text is set off with the ` on either side. Just to be clear. Same comment for below.